lab04_example/src/key_to_cmd_vel_server.cpp

```diff
@@ -0,0 +1,76 @@
+/* http://wiki.ros.org/ROS/Tutorials/WritingServiceClient%28c%2B%2B%29#roscpp_tutorials.2FTutorials.2FWritingServiceClient.Writing_a_Service_Node */
+
+/* Author: Luke Dennis l.dennis@unsw.edu.au */
+
+#include <ros/ros.h>
+#include <lab04_example/KeyToCmdVel.h>
+#include <lab04_example/key_control.h>
+
+
+class KeyCmdSrv {
```

Comment:
Document how this class relates to all other classes

Reply:
it doesn't really

Reply:
58e65d4812f3cc50574b7ac4a35970ee90ad6879ad198a97ad8a38220a446a62

Reply:
c9fd7ca22097d64d394a8c2c00a58c7ac7ab6f7b